Microsoft.Toolkit.Uwp.SampleApp/Controls/SampleAppMarkdownRenderer.cs

```diff
@@ -10,12 +10,14 @@
 // THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
```

Comment:
Doc updates for the control for this new behavior?

I.e. saying that this will happen only if the two blocks are adjacent with no other text between them.  Does extra whitespace matter?

Reply:
I have updated `.template.md` talking about it.

The MarkdownTextBlock doesn't mind if there are spaces between, as long as there is no other content.

I'm not sure about `docs.microsoft.com`. @mrlacey does having whitespace matter? 
Or does it just remove Code Blocks if they don't contain the language specified in the picker?

Reply:
I don't know about whitespace on docs.microsoft.com and can't test it. @nmetulev can you test the impact of having whitespace between code blocks of different languages?